lib/resources/example_json/dstu2_examples_practitioner.rb

```diff
@@ -0,0 +1,77 @@
+module Cerner
+  module Resources
+
+    DSTU2_PRACTITIONER_ENTRY ||= {
+      "resourceType": "Practitioner",
+      "id": "1478007",
```

Comment:
Use a different example (in both) - one of the docs/users we've built specifically for data entry that has most of the fields we return filled out.